app/models/provisioning_template.rb

```diff
@@ -213,3 +183,9 @@ def self.pxe_default_combos
     combos
   end
 end
+
+ConfigTemplate = begin
```

Comment:
Is this to maintain plugin compatibility? If so, I think it'd be fine to remove, only nightly would break at plugins which is fine. If a plugin wants to be compatible with Foreman 1.9 and Foreman 1.8 and they use `ConfigTemplate`, they can do this in their engine initialization for instance.